lib/mp4/transmuxer.js

```diff
@@ -987,14 +985,12 @@ Transmuxer = function(options) {
     // !!THIS ORDER IS IMPORTANT!!
     // demux the streams
     pipeline.elementaryStream
-      .pipe(pipeline.videoTimestampRolloverStream)
+      .pipe(pipeline.timestampRolloverStream)
```

Comment:
This should be moved up to line 984 and then the `pipeline.elementaryStream` piping into each type here should be changed to `pipeline.timestampRolloverStream`